code/game/objects/structures/crates_lockers/closets/bodybag.dm

```diff
@@ -50,21 +51,28 @@
 	if (tagged)
 		add_overlay("bodybag_label")
 
+/obj/structure/closet/body_bag/open(mob/living/user)
+	. = ..()
+	if(.)
+		mouse_drag_pointer = MOUSE_INACTIVE_POINTER
+
 /obj/structure/closet/body_bag/close()
-	if(..())
+	. = ..()
+	if(.)
 		density = FALSE
-		return 1
-	return 0
+		mouse_drag_pointer = MOUSE_ACTIVE_POINTER
 
 /obj/structure/closet/body_bag/MouseDrop(over_object, src_location, over_location)
 	. = ..()
 	if(over_object == usr && Adjacent(usr) && (in_range(src, usr) || usr.contents.Find(src)))
 		if(!ishuman(usr))
-			return 0
+			return
 		if(opened)
-			return 0
+			to_chat(usr, "<span class='warning'>You wrestle with [src], but it won't fold while unzipped.</span>")
```

Comment:
```suggestion
			to_chat(usr, "<span class='warning'>You wrestle with \the [src], but it won't fold while unzipped.</span>")
```

Reply:
wait does src default to `a` or `the` for impropers

Reply:
`[x]` defaults to being the same as `\the [x]`